src/lib/y2partitioner/widgets/lvm_devices_table.rb

```diff
@@ -34,7 +34,8 @@ class LvmDevicesTable < ConfigurableBlkDevicesTable
       # @param devices [Array<Y2Storage::Lvm_vg, Y2Storage::Lvm_lv>] devices to display
       # @param pager [CWM::Pager] table have feature, that double click change content of pager
       #   if someone do not need this feature, make it only optional
-      def initialize(devices, pager)
+      # @param buttons_set [DeviceButtonsSet]
```

Comment:
please document also why it is needed.

Reply:
Actually, that whole documentation should not be there, in `LvmDevicesTable`, which is not the base class. `#devices` and `#pager` are already (better) documented in the base class.